crates/bevy_render/src/color/mod.rs

```diff
@@ -487,6 +487,92 @@ impl Color {
             } => [hue, saturation, lightness, alpha],
         }
     }
+
+    /// Converts a `Color` to a `u32` from sRGB colorspace.
```

Comment:
Would it makes sense to describe how the colour is converted to u32 (each channel as a u8, order as alpha-blue-green-red) or is it standard enough to do it that way?

Reply:
I think it should be described. Perhaps both endianness (it wasn’t obvious to me a while ago that GPUs were generally little endian) and order.

Reply:
Should I change the comment to the following?
Converts `Color` to a `u32` from sRGB colorspace, mapping the RGBA channels in this order to a little-endian byte array (meaning A will be the most significant byte and R the least significant).

Reply:
Sounds good to me.

Reply:
```suggestion
    /// Converts Color to a u32 from sRGB colorspace
    ///
    /// Maps the RGBA channels in RGBA order to a little-endian byte array (GPUs are little-endian).
    /// A will be the most significant byte and R the least significant.
```

Reply:
> A will be the most significant byte

that took me two reads to understand, with "will" being also a noun 😄